.github/workflows/build.yml

```diff
@@ -62,21 +64,53 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          # TODO(hawflau): investigate and fix flakiness on Windows
-          # - windows-latest
+          - windows-latest
```

Comment:
Did the Windows flakiness get resolved?

Reply:
I've confirmed with @hawflau the failures are coming from the test runs (not during tooling setup). See example: https://github.com/aws/aws-sam-cli/actions/runs/4207102433/jobs/7301465033

And rust tests are configured to skip windows CI environment at the moment.